lib/ActivityStream.jsm

```diff
@@ -222,6 +222,10 @@ const PREFS_CONFIG = new Map([
       layout_endpoint: "https://getpocket.com/v3/newtab/layout?version=1&consumer_key=40249-e88c401e1b1f2242d9e441c4&layout_variant=basic",
     }),
   }],
+  ["darkModeMessage", {
+    title: "Boolean flag that decides whether to show the dark Mode message or not.",
+    value: true,
+  }],
```

Comment:
@k88hudson this pref is true for nightly en-US locale only and stay false or undefined for rest of the channels, is there another place where we set that? thanks!

Reply:
You could basically set this to `IS_NIGHTLY_OR_UNBRANDED_BUILD`, so that it would only be true if the build is nightly (and default `false` otherwise)